src/views/search/search.jsx

```diff
@@ -138,4 +144,12 @@ var Search = injectIntl(React.createClass({
     }
 }));
 
-render(<Page><Search /></Page>, document.getElementById('app'));
+var mapStateToProps = function (state) {
+    return {
+        navigationDetails: state.navigationDetails
+    };
+};
+
+var ConnectedDetails = connect(mapStateToProps)(Search);
```

Comment:
Another naming convention small thing – since this is the search page, it should be called `ConnectedSearch`